charts/icm-as/values.yaml

```diff
@@ -84,6 +85,17 @@ fullnameOverride: ""
 # Set a custom server name. Defaults to 'appserver'
 # serverName: <customServerName>
 
+# operational context (~= environment) this icm is running in
+operationalContext:
+  #   id of the customer e.g. edexx
+  # customerId:
+  #   type of the environment (one of {int,uat,prd}, default=prd)
+  # environmentType: prd
```

Comment:
is the default of environment name the environment type?
Why are the defaults not defined at this values.yaml?